### PR TITLE
Docs: add references for 3D–0D coupling examples and svZeroDSolver developer guide

### PR DESCRIPTION
**Pull request by taeoukkim**
This PR adds direct references to help users understand and use 3D–0D coupling:

- A concrete svMultiPhysics 3D–0D coupling example (pipe_RCR_sv0D)
- The svZeroDSolver developer guide for creating custom 0D blocks

Updated section:
documentation/multi_physics.html#svzerodsolver_interface_parameters

### CODE REVIEW

File: documentation/multi_physics/solver-input-file/svzerodsolver_interface_parameters/readme.md [hunk 1]
```diff
@@ -7,6 +7,22 @@ The <i>svZeroDSolver Interface Subsection</i> of the <i>Equation Section</i> def
...
 
+<p>
+For a concrete example of how to set up and use 3D–0D coupling in practice, see the
+<a href="https://github.com/SimVascular/svMultiPhysics/tree/main/tests/cases/fluid/pipe_RCR_sv0D" target="_blank">
+svMultiPhysics 3D–0D coupling example case
+</a>.
+</p>
+
```
Comment (ktbolt) [inline documentation/multi_physics/solver-input-file/svzerodsolver_interface_parameters/readme.md:17]: @taeoukkim Replace the above with

<p>
See the <a href="https://github.com/SimVascular/svMultiPhysics/tree/main/tests/cases/fluid/pipe_RCR_sv0D"> pipe_RCR_sv0D </a> svMultiPhysics test case for an example of using the svZeroDSolver interface for an RCR boundary condition in a simple fluid simulation.

File: documentation/multi_physics/solver-input-file/svzerodsolver_interface_parameters/readme.md [hunk 2]
```diff
@@ -7,6 +7,22 @@ The <i>svZeroDSolver Interface Subsection</i> of the <i>Equation Section</i> def
...
+<p>
+If you would like to create your own custom 0D blocks, please refer to the
+<a href="https://simvascular.github.io/svZeroDSolver/developer_guide.html" target="_blank">
+svZeroDSolver developer guide
+</a>.
+</p>
```
Comment (ktbolt) [inline documentation/multi_physics/solver-input-file/svzerodsolver_interface_parameters/readme.md]: @taeoukkim Remove this, don't want to talk about 0D blocks here.

File: documentation/multi_physics/solver-input-file/svzerodsolver_interface_parameters/readme.md (hunk 1 above)
Comment (taeoukkim) [reply documentation/multi_physics/solver-input-file/svzerodsolver_interface_parameters/readme.md:17]: I replaced with your suggestion!

File: documentation/multi_physics/solver-input-file/svzerodsolver_interface_parameters/readme.md (hunk 2 above)
Comment (taeoukkim) [reply documentation/multi_physics/solver-input-file/svzerodsolver_interface_parameters/readme.md]: I removed this with your suggestion!

**Review by ktbolt — APPROVED** [review]
@taeoukkim Thanks for the updates !
